src/mjolnir/graphbuilder.cc

```diff
@@ -1250,7 +1259,11 @@ std::vector<SignInfo> GraphBuilder::CreateExitSignInfoList(const OSMNode& node,
     std::vector<std::string> branch_streets =
         GetTagTokens(osmdata.name_offset_map.name(way.destination_street_index()));
     for (auto& branch_street : branch_streets) {
-      exit_list.emplace_back(Sign::Type::kExitBranch, false, branch_street);
+      if (!link) {
```

Comment:
There are exit signs that are not on links - for example: https://www.openstreetmap.org/way/6009977#map=16/39.3222/-76.5308

Reply:
a link = turn channel or ramp